ledger/src/blockstore_db.rs

```diff
@@ -1450,9 +1486,28 @@ fn get_cf_options<C: 'static + Column + ColumnName>(
         });
     }
 
+    process_cf_options_advanced::<C>(&mut options, advanced_options);
+
     options
 }
 
+fn process_cf_options_advanced<C: 'static + Column + ColumnName>(
+    cf_options: &mut Options,
+    advanced_options: &BlockstoreAdvancedOptions,
+) {
+    if is_cf_compression_enabled::<C>() {
+        cf_options.set_compression_type(
+            advanced_options
+                .compression_type
+                .to_rocksdb_compression_type(),
+        );
+    }
+}
+
+fn is_cf_compression_enabled<C: 'static + Column + ColumnName>() -> bool {
```

Comment:
nit: `should_enabled_cf_compression()` would match the other similar functions (`should_dsiable_auto_compaction()` and `should_exclude_from_compaction()`)

Also, this function is similar to those in terms of purpose, maybe move this function (or move the other two) such that they are all sequential in the file